addons/ITCExpansionSpike/functions/fn_reloaded.sqf

```diff
@@ -0,0 +1,5 @@
+/*
+ * itc_land_spike_fnc_reloaded
+ */
+params ["_unit", "_weapon", "_muzzle", "_newMagazine", "_oldMagazine"];
```

Comment:
Fehlt hier was oder wofür ist das file da?

Reply:
das mit den functions is noch net fertig, der version bump war nur fürs testen